components/icons/NavbarIcons.tsx

```diff
@@ -102,18 +102,23 @@ export const NowcastingLogo: React.FC = ({}) => (
 
 export const MenuLogo: React.FC = ({}) => (
   <svg
-    xmlns="http://www.w3.org/2000/svg"
+    width="88"
+    height="24"
+    viewBox="0 0 88 24"
     fill="none"
-    viewBox="0 0 24 24"
-    strokeWidth={1.5}
-    stroke="currentColor"
-    color="#FFD053"
-    className="w-10 h-10"
+    xmlns="http://www.w3.org/2000/svg"
+    className="w-20 h-20"
   >
     <path
-      strokeLinecap="round"
-      strokeLinejoin="round"
-      d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
+      d="M2.25 21H21.75M3.75 3V21M14.25 3V21M20.25 7.5V21M6.75 6.75H7.5M6.75 9.75H7.5M6.75 12.75H7.5M10.5 6.75H11.25M10.5 9.75H11.25M10.5 12.75H11.25M6.75 21V17.625C6.75 17.0037 7.25368 16.5 7.875 16.5H10.125C10.7463 16.5 11.25 17.0037 11.25 17.625V21M3 3H15M14.25 7.5H21M17.25 11.25H17.2575V11.2575H17.25V11.25ZM17.25 14.25H17.2575V14.2575H17.25V14.25ZM17.25 17.25H17.2575V17.2575H17.25V17.25Z"
+      stroke="white"
+      stroke-width="1.5"
+      stroke-linecap="round"
+      stroke-linejoin="round"
```

Comment:
think these should be changed too 
```suggestion
      strokeWidth="1.5"
      strokeLinecap="round"
      strokeLinejoin="round"
```